components/button/style/index.less

```diff
@@ -142,9 +142,7 @@
   }
 
   &-background-ghost {
-    background: transparent!important;
```

Comment:
I do a default (white) button on a white background... and i already define primary ghost button with blue color